src/react-components/src/components/02-molecules/Headings/SectionTitle.tsx

```diff
@@ -30,9 +30,9 @@ export default class SectionTitle extends React.Component<SectionTitleProps, {}>
     const baseClass = "heading-section";
 
     if (headingText.length > 80) {
-      throw new Error("Section Title Text must be fewer than 80 characters");
+      throw new Error("Section Title (h2) Text must be fewer than 80 characters");
     } else if (headingText.length > 60) {
-      console.warn("Section Title Text should be fewer than 60 characters");
+      console.warn("Section Title (h2) Text should be fewer than 60 characters");
```

Comment:
I think I better understand the whole DS system a little better now. A developer would be checking multiple documentation in metronome as well as this React library, right? That way they know that `SectionTitle` will only render an h2, right?

Reply:
Yes. Developers will receive Metronome documentation that will (in the future) have Storybook embeds of the relevant stories for both the Twig and React implementation of the component. Those pages have the requirements that the component was built to.